pkg/controller/replication/replication_controller.go

```diff
@@ -212,6 +223,13 @@ func (rm *ReplicationManager) getPodController(pod *api.Pod) *api.ReplicationCon
 	// pod: [(k1:v1), (k2:v2)] will wake both rc1 and rc2, and we will sync rc1.
 	// pod: [(k2:v2)] will wake rc2 which creates a new replica.
 	sort.Sort(overlappingControllers(controllers))
+	// Surface that we are ignoring multiple controllers for a single pod
+	for i, _ := range controllers {
+		if i == 0 {
+			continue
+		}
+		rm.reportConflict(&controllers[i], &controllers[0])
```

Comment:
How does this enforce which controller to mark as conflicting?  It seems there is no guarantee it would pick newer or older controller?  Also how do you know the user intent on which to mark conflicting?